streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -3010,13 +2993,9 @@ public void initializeIfNeeded() {
             }
         };
 
-        consumer.commitSync(Collections.emptyMap());
-        expectLastCall();
-        expect(consumer.assignment()).andReturn(emptySet());
-        consumer.resume(eq(emptySet()));
-        expectLastCall();
```

Comment:
Same as above, Mockito reported these are unused. I added an assertion in Mockito world at the end (...I should probably add one in the first test as well, but I can do this in part 2 of this pull request)